### PR TITLE
Modify advfilter

### PR DESCRIPTION
**Pull request by DiegoTheExplorar**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AdvFilterCommand.java [hunk 1]
```diff
@@ -13,9 +13,9 @@
 /**
  * Sort the contact list based on tag names and values.
  */
-public class SortCommand extends Command {
+public class AdvFilterCommand extends Command {
```
Comment (valhrd) [inline src/main/java/seedu/address/logic/commands/AdvFilterCommand.java:17]: LGTM

File: src/main/java/seedu/address/logic/commands/AdvFilterCommand.java [hunk 2]
```diff
@@ -13,9 +13,9 @@
...
 
-    public static final String COMMAND_WORD = "sort";
+    public static final String COMMAND_WORD = "advfilter";
```
Comment (valhrd) [inline src/main/java/seedu/address/logic/commands/AdvFilterCommand.java:19]: Nice change!

File: build.gradle [hunk 3]
```diff
@@ -69,4 +69,8 @@ shadowJar {
     archiveFileName = 'tp.jar'
 }
 
+run {
+    enableAssertions = true
```
Comment (valhrd) [inline build.gradle:73]: Thanks for this!

File: src/main/java/seedu/address/logic/commands/AdvFilterCommand.java [hunk 4]
```diff
@@ -87,11 +87,11 @@ public boolean equals(Object other) {
             return true;
         }
 
-        if (!(other instanceof SortCommand)) {
+        if (!(other instanceof AdvFilterCommand)) {
             return false;
         }
 
-        SortCommand otherCommand = (SortCommand) other;
+        AdvFilterCommand otherCommand = (AdvFilterCommand) other;
```
Comment (valhrd) [inline src/main/java/seedu/address/logic/commands/AdvFilterCommand.java:94]: This looks safe!

File: src/main/java/seedu/address/logic/parser/AdvFilterCommandParser.java [hunk 5]
```diff
@@ -5,27 +5,27 @@
...
+import seedu.address.logic.commands.AdvFilterCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 /**
  * Parses input for sort and creates an instance of SortCommand
  */
-public class SortCommandParser implements Parser<SortCommand> {
+public class AdvFilterCommandParser implements Parser<AdvFilterCommand> {
     //regex to filter the input to extract TAG name, comparison operator and TAG value
-    private static final Pattern SORT_COMMAND_FORMAT =
+    private static final Pattern ADVFILTER_COMMAND_FORMAT =
```
Comment (valhrd) [inline src/main/java/seedu/address/logic/parser/AdvFilterCommandParser.java:15]: I like this implementation!

**Review by valhrd — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/AdvFilterCommand.java [hunk 6]
```diff
@@ -33,7 +33,7 @@ public class SortCommand extends Command {
     /**
      * Class that handles SortCommand
      */
-    public SortCommand(String tagName, String operator, String tagValue) {
+    public AdvFilterCommand(String tagName, String operator, String tagValue) {
```
Comment (appleshill) [inline src/main/java/seedu/address/logic/commands/AdvFilterCommand.java:36]: Elegant solution!

File: src/main/java/seedu/address/logic/parser/AdvFilterCommandParser.java [hunk 7]
```diff
@@ -5,27 +5,27 @@
...
             Pattern.compile("(?i)(t/(?<tag>\\S+)){1}\\s+(?<operator>\\S+){1}\\s+(?<value>\\S+){1}");
 
     /**
      * Parses the given {@code String} of arguments in the context of the SortCommand
      * and returns a SortCommand object for execution.
      * @throws ParseException if the user input does not conform to the expected format
      */
-    public SortCommand parse(String args) throws ParseException {
-        final Matcher matcher = SORT_COMMAND_FORMAT.matcher(args.trim());
+    public AdvFilterCommand parse(String args) throws ParseException {
```
Comment (appleshill) [inline src/main/java/seedu/address/logic/parser/AdvFilterCommandParser.java:23]: Really like this approach!

File: src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java [hunk 8]
```diff
@@ -25,7 +25,7 @@
 import seedu.address.model.UserPrefs;
 import seedu.address.model.person.Person;
 
-public class SortCommandTest {
+public class AdvFilterCommandTest {
```
Comment (appleshill) [inline src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java:28]: Excellent refactor

**Review by appleshill — APPROVED** [review]

**Pull request merged**